enhancements/observability/multicluster-observability-addon.md

```diff
@@ -0,0 +1,767 @@
+
+### Test Plan
+
+TBD
+
+### Graduation Criteria
+
+TBD
+
+#### Dev Preview
+
+- Installation of the ClusterLogForwarder and OpenTelemetryCollector CRs.
+- Support for all the authentication methods supported by ClusterLogForwarder (i.e. static auth, mTLS auth, etc).
+- Support for Per-ClusterSet ClusterLogForwarder.
+- Health probing per Signal
+- Status reporting
+
+#### Dev Preview -> Tech Preview
+
+- Refined CLF/OTelCollector CRs usage
+
+#### Tech Preview -> GA
+
+TBD
+
+#### Removing a deprecated feature
+
+None
+
+### Upgrade / Downgrade Strategy
+
+None
+
+### Version Skew Strategy
+
+None
+
+### Operational Aspects of API Extensions
+
+TBD
+
+#### Failure Modes
+
+- Cert Manager not installed: this will not allow the addon to create it's own CA and provision mTLS certificates
+- Missing Stanza: if a ManagedClusterAddon references a stanza that doesn't exist this will block the reconciliation loop as the addon will not be able to generate a `ManifestWorks` 
+- Incorrect signal labels: if the user incorrectly sets the signal label `mcoa.openshift.io/signal` on resources that are necessary these resources will not be used for configuration.
+- Missing Secrets: if an output uses a secret provided by the user if that secret is missing then this will block the reconciliation loop;
+- Wrong key in the CA resouce: if the users provide a resource with the CA data but define the wrong key (not `service-ca.crt`) then this will block the reconciliation
+
+
+#### Support Procedures
+
+TBD
+
+## Implementation History
+
+|      | PoC | Demo | DevPreview | TechPreview | General Availability |
+|------|-----|------|------------|-------------|----------------------|
+| MCOA |  ✅  |   ✅  |      🚧     |             |                      |
+
+POC - https://github.com/ViaQ/logging-ocm-addon
+Demo - https://github.com/rhobs/multicluster-observability-addon/tree/main/demo
+DevPreview - https://github.com/rhobs/multicluster-observability-addon
+
+## Alternatives
+
+### Policy-Engine based workflow
+
+The RHACM policy engine is part of the RHACM governance tools to apply certain policies on workloads and infrastructure on the entire fleet. It serves well to a degree however for the MCOA use-case it falls short since MCOA will need to:
+- manage complext dependencies between multiple components, OpenShift Logging Operator installation (i.e Cluster-Logging-Operator) and the custom resources (i.e. ClusterLogForwarder) as well as authentication related artifacts (i.e. generating TLS certificates, Cloud Provider Managed Identities / ServiceAccounts, etc.).
+- build dynamic error reporting (e.g. operator installation/upgrade failures, CRD handling) and health reporting (e.g. running/ready progress of operators and operands).
+
+The spike for this alternative can be found here: https://gitlab.cee.redhat.com/openshift-logging/log-storage-toolbox/-/merge_requests/14
+
+### Integrate directly into present Multi-Cluster-Observability-Operator
```

Comment:
This is the final decision, right?

Reply:
This alternative relates to the idea of instead creating MCOA just updating the MCO code directly.